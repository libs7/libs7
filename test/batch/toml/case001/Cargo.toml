# src: https://raw.githubusercontent.com/rhzx86/cargo-examples/main/Cargo.toml

[package]
name = "cargo-examples"
version = "0.5.0"
authors = ["Richard Hoz√°k"]
edition = "2021"
license = "MIT OR Apache-2.0"
readme = "README.md"
homepage = "https://github.com/zxey/cargo-examples"
description = "Run all examples for any locally cloned crate"
categories = [
    "command-line-utilities",
    "development-tools::cargo-plugins",
]
keywords = [
    "cargo",
    "cargo-subcommand",
    "cli",
    "examples",
    "crates",
]

[dependencies]
anyhow = "1.0.65"
clap = { version = "4.0", features = ["derive", "foo"] }
tap = "1.0.1"
xshell = "0.2.2"
cargo_toml = "0.14.0"