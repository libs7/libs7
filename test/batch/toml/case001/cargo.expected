Package
  Name:        cargo-examples
  Version:     0.5.0
  Authors:
      Richard Hoz√°k
  Edition:     2021
  License:     MIT OR Apache-2.0
  Readme:      README.md
  Homepage:    https://github.com/zxey/cargo-examples
  Description: Run all examples for any locally cloned crate
  Categories:
      command-line-utilities
      development-tools::cargo-plugins
  Keywords:
      cargo
      cargo-subcommand
      cli
      examples
      crates

Dependencies:
    anyhow      :   1.0.65
    tap         :    1.0.1
    xshell      :    0.2.2
    cargo_toml  :   0.14.0
    clap        : version'4.0'features&quot;derive&quot;&quot;foo&quot;

NB: support for ad-hoc nested destructuring (in this case, 'clap') is
under development. In the meantime, we must know the structure ahead
of time to destructure:

clap:
    version:  4.0
    features: derive, foo
